lib/pacemaker/pcmk_options.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+
+#include <pacemaker.h>
+#include <pacemaker-internal.h>
+
+/*!
+ * \internal
+ * \brief List all available cluster options
+ *
+ * These are options that affect the entire cluster.
+ *
+ * \param[in,out] out  Output object
+ *
+ * \return Standard Pacemaker return code
+ */
+int
+pcmk__list_cluster_options(pcmk__output_t *out)
+{
+    const char *name = "cluster-options";
+    const char *desc_short = "Pacemaker cluster options";
+    const char *desc_long = NULL;
+
+    desc_long = "Also known as properties, these are options that affect "
```

Comment:
arguably too verbose, but erred on the side of being friendly to newbies running list-options commands. not tied to it

Reply:
On second thought, ignore that. It could be a significant project on its own (created T776).

Reply:
The proposal in T776 seems better and more general, but it seems like it would be pretty straightforward to have `--long-desc` and `--short-desc` options that go with `--list-options`

Reply:
I'd rather not be stuck with maintaining them. The command-line option approach won't scale as we add more, and it's harder for higher-level tools to discover. We can live with the status quo.

Reply:
Agreed.